hearts/game/hearts.py

```diff
@@ -496,6 +496,8 @@ def serialize(self, for_player=None):
             'direction': str,
             'pass_selections': {str: [str, str, str]}
             'turn': int,
+            'player_action': {str: str},
+            'messages': {str: [str]},
```

Comment:
I assumed that the messages and actions would be on the serialized game object, not the round, so my front end code will be broken for this. If you want you can leave it how it is and change the front end, or move this to the base game.

I think it makes sense either way.

Reply:
I'll leave it the way it is and try to change the frontend to work.  Right now, when four players join a game, the `RoomUI` does not display my hand.